smartbms.py

```diff
@@ -131,9 +128,11 @@ def __init__(
         self.firmware_version = None
         self.charge_cycles_guard = [KeyValuePairGuard(), KeyValuePairGuard()] # Contains high and low byte
         self.charge_cycles = None
-        self.charged_energy_guard = [KeyValuePairGuard(), KeyValuePairGuard(), KeyValuePairGuard()] # Contains 3 bytes
+        self.charged_energy_guard = [KeyValuePairGuard(), KeyValuePairGuard(),
+                                     KeyValuePairGuard()] # Contains 3 bytes
```

Comment:
Did this split into multiple lines and if so, why exactly?

Reply:
Missed this one, you're correct. Will amend.